test/es6/module_4482_dep2.js

```diff
@@ -4,10 +4,10 @@
 //-------------------------------------------------------------------------------------------------------
 
 //thing1.js
-import { Thing2 } from './module_4482_dep1.js';
```

Comment:
Not super familiar with this bug, but is removing the ./ relevant or important?

Reply:
The "./" isn't important, I just cleaned it up whilst doing this.